browser/main/NewNoteButton/index.js

```diff
@@ -21,6 +21,9 @@ class NewNoteButton extends React.Component {
     this.state = {
     }
 
+    this.handleNewNoteButtonClick = this.handleNewNoteButtonClick.bind(this)
+
+    // why is there a newNoteHandler?!
```

Comment:
I think we don't need it because we have `this.handleNewNoteButtonClick`. But we have to fix `componentDidMount` and `componentWillUnmount ` also so they can use `this.handleNewNoteButtonClick` too.

Reply:
OK, removed the not needed `newNoteHandler` and updated the mentioned locations.